plop-templates/Component/Component.tsx.hbs

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from "react";
+
+import classNames from "classnames/bind";
+import styles from "./{{pascalCase name}}.module.css";
+
+const cx = classNames.bind(styles);
+
+type Props = {
+  /**
+   * Child node(s) that can be nested inside component.
+   */
+  children: ReactNode;
+  /**
+   * How the text should be aligned.
+   */
+  align: "left" | "right";
+};
+
+/**
+ * This component is a replica of the template created by plop found in plop-templates.
+ * TODO: Find a way to keep these 2 files in sync.
```

Comment:
what does this mean?

Reply:
Oh this is from before I talked to you about the purpose of the `PlopTest` component in your PR. I'll remove this.